py-polars/polars/utils/convert.py

```diff
@@ -38,6 +25,29 @@
     P = ParamSpec("P")
     T = TypeVar("T")
 
+    # the below shenanigans with ZoneInfo are all to handle a
+    # typing issue in py < 3.9 while preserving lazy-loading
```

Comment:
There is actually some movement on that issue now, hopefully a new backports.zoneinfo release that fixes the issue :)

https://github.com/pganssle/zoneinfo/pull/126

Reply:
Nearly there... zoneinfo and py3.7 dealt with; one issue with py3.11 to go...